mongodbforms/documents.py

```diff
@@ -8,7 +8,10 @@
 from django.forms.widgets import media_property
 from django.core.exceptions import FieldError
 from django.core.validators import EMPTY_VALUES
-from django.forms.util import ErrorList
+try:
+    from django.forms.utils import ErrorList
+except ImportError:
```

Comment:
this is kind of harsh. But maybe we can treat this in a separate task ? (I mentioned in the PR desc that I didn't remove one deprecation warning that was difficult to cleanup, maybe we can do it at that time)